support-frontend/assets/components/thankYou/thankYouModuleData.tsx

```diff
@@ -0,0 +1,70 @@
+import Button from 'components/button/button';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import AppDownloadBadges from './appDownload/AppDownloadBadges';
+import type { ThankYouModuleType } from './thankYouModule';
+import { getThankYouModuleIcon } from './thankYouModuleIcons';
+
+type ThankYouModuleData = {
+	icon: JSX.Element;
+	header: string;
+	bodyCopy: string;
+	ctas: JSX.Element;
+};
+
+export const getThankYouModuleData = (
```

Comment:
This function is not yet in use, it will be used on the V2 Thank You page to render all modules